master/static/srv/check_idle.py

```diff
@@ -57,21 +79,20 @@ def main():
         )
         idle_type = IdleType.KERNELS_OR_TERMINALS
 
-    while True:
-        sleep(1)
+    wait_for_jupyter(("127.0.0.1", int(port)))
```

Comment:
127.0.0.1 ???

Reply:
Yes, this is inside the container, and jupyter is also inside the container.

`127.0.0.1` is used for the notebook_server address as well.